test/e2e/react-client--scr.test.js

```diff
@@ -44,9 +44,11 @@ describe('test:e2e:react:scr', () => {
       JSON.stringify(answers.entityManagement3), '../../');
 
     await checkFormat(SCR_APP_DIR);
-    await installAndBuild('scr-model', SCR_APP_DIR);
 
-    await runTests(SCR_APP_DIR);
+    //todo uncomment after merge
+    //disabled due to publishing new version of cuba-react
+    // await installAndBuild('scr-model', SCR_APP_DIR);
+    // await runTests(SCR_APP_DIR);
```

Comment:
I would prefer to avoid merging disabled tests to master. I would suggest the following flow for now (until we have a better process with monorepo):

- create PRs for `cuba-react` and `front-generator`. `front-generator` tests will fail at this point due to lack of corresponding `cuba-react` version in npm.
- `cuba-react` is merged and published as dev version. `front-generator` tests no longer fail.
- `front-generator` is merged
- once we are ready for release we change the dev version to normal version